infra/charts/turing/templates/_helpers.tpl

```diff
@@ -101,6 +101,12 @@ MLPConfig:
 TuringEncryptionKey: {{ include "turing.encryption.key" . | quote }}
 Sentry:
   DSN: {{ .Values.turing.config.Sentry.DSN | default (include "turing.sentry.dsn" .) | quote }}
+OpenapiConfig:
+  ValidationEnabled: true
+  SwaggerUIConfigOverrideFile: /etc/openapi/override.yaml
+  SwaggerUIConfig:
+    ServingDirectory: swagger-ui
+    ServingPath: /api-docs/
```

Comment:
Having the default values only here is not enough, as it doesn't address a situation, when turing-api isn't deployed from the chart, but is running locally or in a docker container (for development/test purposes)

Reply:
Using this as the central discussion point: https://github.com/gojek/turing/pull/140#discussion_r760959454